.github/workflows/ci.yaml

```diff
@@ -89,8 +101,14 @@ jobs:
     container: akmetiuk/dotty:2020-02-12
 
     steps:
-      - name: Clean Workspace
-        uses: anatoliykmetyuk/action-clean@1.0.2
+      - name: Cleanup
+        run: |
+          TARGETS="$(pwd) /__w/_temp /github/home"
+          for folder in $TARGETS; do
+            echo "Cleaning $folder"
+            cd $folder
+            rm -rf ..?* .[!.]* *
```

Comment:
How about:
```suggestion
            rm -rf $folder
            mkdir $folder
```

Reply:
I think the original version works well enough: it cleans up both hidden and non-hidden files. The problem with deleting the folders is that if they are mounted by Docker, we might get the "device is busy error".

Reply:
We should probably factor this out in a script

Reply:
It's tricky with the current setup. CIs are surprisingly bad at modularisation, neither Drone nor GH Actions doesn't allow to save steps under a name and then refer to them later. I'll merge this FTTB since this is quite high prio.